internal/openshift/controller.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/tenant"
 	"github.com/fabric8-services/fabric8-jenkins-idler/internal/toggles"
 	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/watch"
```

Comment:
I believe that If you aAre you adding this you need to modify the gopkg.toml too, or regenerate the lock file